testData/solutions/win32.json5

```diff
@@ -3724,6 +3724,85 @@
             }
         ]
     },
+    "com.microsoft.windows.desktopBackground": {
+        "name": "Windows desktop background personalization",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configure": {
+                "type": "gpii.windows.desktopBackgroundSettingsHandler",
+                "liveness": "live",
+                "options": {},
+                "supportedSettings": {
+                    "BackgroundType": {
+                        "schema": {
+                            "title": "Desktop background payload",
+                            "description": "The type of background that user wants to select",
+                            "type": "string",
+                            "default": "Picture",
+                            "enum": [
+                                "SolidColor",
+                                "Picture",
+                                "Slideshow"
+                            ]
+                        }
+                    },
+                    "Input": {
+                        "schema": {
+                            "title": "Desktop background payload",
+                            "description": "The values that user want to use for that particular background type",
+                            "type": "Object",
+                            "default": {
+                                "Image": "‪C:\\Windows\\Web\\Wallpaper\\Windows\\img0.jpg",
+                                "Scaling": "Fill"
+                            },
+                            "enum": [
+                                // SolidColor 'BackgroundType' Input payload format
+                                {
+                                    "Color": "{ r: 0..0xff, g: 0..0xff, b: 0..0xff }"
+                                },
+                                // Slideshow 'BackgroundType' Input payload format
+                                {
+                                    "Shuffle": "True..False"
```

Comment:
boolean?

Reply:
We should be quite careful and verify what the program actually stores and expects to see in the actual setting.  For NVDA, for example, they were storing literal "True" and "False" strings, so using a boolean and relying on its toString method was not quite correct, or at least didn't match what NVDA itself did.  Verifying that "true" and "false" actually work in context and then using `type: "boolean"` is fine.

Reply:
Just to clarify, the syntax here is not at all correct, just saying that one of the outcomes could be an enum of "True" and "False".  Search for instances of ".." throughout, that shorthand is not useful in this context.